AppController/lib/zkinterface.rb

```diff
@@ -501,6 +501,15 @@ def self.get_version_details(project_id, service_id, version_id)
   end
 
 
+  # Defines a deployment-wide default for maximum instance memory.
+  def self.set_max_memory(max_memory)
+    max_memory_node = '/appscale/config/max_memory'
+    data = {'max_memory' => max_memory}
```

Comment:
I agree that it's redundant, but I would like to keep it consistent with the other nodes in that path. Our DeploymentConfig watch assumes that all configuration nodes are formatted as a JSON dictionary. Do you think I should rename the node to something like /appscale/config/runtime_parameters in case we add something other than max_memory?